evidence/doc.go

```diff
@@ -0,0 +1,37 @@
+/*
+Package evidence handles all evidence storage and gossiping from detection to block proposal.
+For the different types of evidence refer to the `evidence.go` file in the types package
+or https://github.com/tendermint/spec/blob/master/spec/consensus/light-client/accountability.md.
+
+## Gossiping
```

Comment:
Godoc sadly doesn't support markdown or any formatting other than indenting for pre-formatted text: https://blog.golang.org/godoc 

I think it would be best to just delete the headings with "##"; I think this whole thing parses fine without them.